lib/tree-view.coffee

```diff
@@ -765,3 +784,60 @@ class TreeView extends View
   # Returns boolean
   multiSelectEnabled: ->
     @list[0].classList.contains('multi-select')
+
+  onDragEnter: (e) ->
+    e.stopPropagation()
+
+    e.currentTarget.parentNode.classList.add('selected')
+
+  onDragLeave: (e) ->
+    e.stopPropagation()
+
+    e.currentTarget.parentNode.classList.remove('selected')
+
+  # Public: Handle entry name object dragstart event
```

Comment:
I just made them public because all other callbacks were documented as public as well, e.g. `resizeStarted`, `entryClicked`, `onMouseDown`. I can refactor this whole think if you like